Description: Do not upload wallpaper after updating or login
 * This behaviour may be unexpected for a user and cause to leak
   private pictures to the remote server.
 * The patch prevents the checkUploadWallPaper() method from being
   called during startup initialization. The user still has an
   opportunity to send his subsequent wallpapers to Telegram Cloud.
   To do this, he can click on the link "Choose from file".
Author: Nicholas Guriev <guriev-ns@ya.ru>
Bug: https://github.com/telegramdesktop/tdesktop/pull/5860
Last-Update: Sat, 27 Jul 2019 14:38:17 +0300

--- a/Telegram/SourceFiles/window/themes/window_theme.cpp
+++ b/Telegram/SourceFiles/window/themes/window_theme.cpp
@@ -556,11 +556,8 @@ void ChatBackground::start() {
 	initialRead();
 
 	Core::App().domain().activeSessionValue(
-	) | rpl::filter([=](Main::Session *session) {
-		return session != _session;
-	}) | rpl::start_with_next([=](Main::Session *session) {
+	) | rpl::start_with_next([=](Main::Session *session) {
 		_session = session;
-		checkUploadWallPaper();
 	}, _lifetime);
 
 	Core::App().settings().setSystemDarkMode(Platform::IsDarkMode());
